Index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+    <head> 
+        <meta charset="UTF-8">
+        <link rel="stylesheet" href="style.css">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">  
+        <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+        <title>My Portfolio</title>
+    </head>
+    <body>
+        <header class="header col-12">
+            <a href="#"><h1>Welcome</h1></a>
+            <nav class="nav-bar">
+                <i class="fa fa-bars" ></i>
+            </nav>
+        </header>
+        <section class="headline col-12">
+
+            <hgroup class="section-heading">
+            <h1>
+                Hey there. I'm Rida
+            </h1>
+            <h2>
+                I'm a software developer
+            </h2>
+        </hgroup>
+        <div class="headline-detail">
+            <p>
+                I can help you build product,
+                feature or website Look through
+                some of my Work and experience!
+                If you like what you see and have 
+                a project you need coded, don't hesitate
+                to contact me.
+            </p>
+
+            <ul>
+                <li><a href="#"><span class="fa fa-twitter"></span></a></li>
+                <li><a href="#"><span class="fa fa-angellist"></span></a></li>
+                <li><a href="#"><span class="fa fa-github"></span></a></li>
+                <li><a href="#"><span class="fa fa-linkedin"></span></a></li>
+            </ul>
```

Comment:
- [ ] _[Optional]_ The social media icons are not well centred.
- [ ] _[Optional]_ The spaces between the social media icons are too much compared to that of the design

Yours | Original
----- | -----
![image](https://user-images.githubusercontent.com/16058470/121345019-45aa7b80-c91c-11eb-9e69-5abef4c5b59b.png) | ![image](https://user-images.githubusercontent.com/16058470/121345102-5ce96900-c91c-11eb-8791-40d2b1d72d18.png)